great-idea/css/index.css

```diff
@@ -63,4 +63,155 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+.container {
+	max-width: 960px;
+	margin: 0 auto;
+}
+
+/* HEADER STYLING */
+.nav-bar {
+	display: flex;
+	justify-content: space-around;
+	margin: 0 0 20px 0;
+	padding: 20px 0;
+	width: 960px;
+	position: fixed;
+	background-color: aliceblue;
+
+}
+
+header a {
+	padding: 20px 20px;
+	text-decoration: none;
+	color: #5c5959;
+}
+
+/* BANNER */
+
+.banner {
+	padding: 20px;
+	display: flex;
+	justify-content: space-around;
+	margin: 80px 0;
+	border-bottom: 1px solid #000;
+}
+
+.banner h1 {
+	width: 50%;
+	text-align: center;
+	font-size: 80px;
+	margin: 0;
+}
+
+button {
+	padding: 15px;
+	background-color: #ffffff;
+	width: 200px;
+	border: 1px solid #000;
+	font-size: 20px;
+}
+.index p {
+	line-height: 20px;
+}
+
+/* MAIN CONTENT */
+.main-content {
+	display: flex;
+	justify-content: space-between;
+	padding: 20px 20px;
+}
+
+.img img {
+	margin: 0 auto;
+	padding: 20px 20px;
+}
+
+main.index {
+	border-bottom: 1px solid #000;
+	padding: 20px 0;
+	margin-top: 0;
+}
+
+/* FOOTER */
+footer {
+	margin: 20px;
+}
+
+footer .contact {
+  line-height: 30px;
+}
+
+footer .copyright {
+	margin: 0 auto;
+	text-align: center;
+	padding: 10px;
+}
+
+/* SERVICES PAGE */
```

Comment:
Good use of comments. Keep them coming :)

Reply:
Thank you!